### PR TITLE
Fixed Javascript undefined error when a saved address is used for validation

### PR DESCRIPTION
**Pull request by erfanimani**

### CODE REVIEW

**Comment by erfanimani** [comment]
Maybe this code is in a different spot on your site? It won't work without it..

**Comment by ProxiBlue** [comment]
Nope. About the same fix. Did not commit it.

File: src/app/design/frontend/base/default/template/fontis/australia/checkout/onepage.phtml [hunk 1]
```diff
@@ -22,6 +22,7 @@
         var australiaCountryRegions = <?php echo $this->helper('directory')->getRegionJson(); ?>;
         var addressSuggestTemplate = <?php echo Zend_Json::encode($this->getChildHtml('fontis.australia.address.suggestion')); ?>;
         var addressFailureTemplate = <?php echo Zend_Json::encode($this->getChildHtml('fontis.australia.address.failure')); ?>;
+        var validateAddressById = parseInt('<?php echo Mage::getStoreConfig('fontis_australia/address_validation/enabled_by_id'); ?>');
```
Comment (erfanimani) [inline src/app/design/frontend/base/default/template/fontis/australia/checkout/onepage.phtml:25]: @ProxiBlue , `addresspotentialsTemplate` is also missing, but that might be specific to your project?

Comment (ProxiBlue) [inline src/app/design/frontend/base/default/template/fontis/australia/checkout/onepage.phtml:25]: Ping me tomorrow. I will check if tgat file exists and include it

**Pull request merged**